tests/test_opt.py

```diff
@@ -299,6 +299,15 @@ def test_merge_common(self):
         self.assertEqual(len(new_nodes), 11)
         self.assertIsNotNone(model)
 
+    def test_onnx_models(self):
+        model_names = ['mobile_segnet_no_opt.onnx', 'srgan_no_opt.onnx']
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        for model_name_ in model_names:
+            model_dir = dir_path + '/data/' + model_name_
+            origin_model = onnx.load_model(model_dir)
+            opt_model = optimize_onnx_model(origin_model)
+            self.assertIsNotNone(opt_model)
```

Comment:
Why don't you let the ORT inference on a test image?

Reply:
(1) mobile_segnet input is a gray image so it needs some preprocessing. srgan we use random input.
(2) These two models have different input sizes
(3) keras2onnx nightly build does the ORT inference.